packages/react/src/component-tests/IcAlert/IcAlert.cy.tsx

```diff
@@ -0,0 +1,344 @@
+/// <reference types="Cypress" />
+import { mount } from "@cypress/react";
+import React from "react";
+import { IcAlert, IcButton, IcLink } from "../../components";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+
+describe("IcAlert", () => {
+  it("Neutral Alert with Message and Icon", () => {
```

Comment:
We could add the visual regression and a11y tests for the Netural alert within this test. It doesn't need to be in a separate describe block. This could be the same for all the other variants as well. Add an visual regression/a11y test for each variant in the relevant tests below.